one_step_setup

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
Thoughts on embedding all of this logic in the `pants` script instead?

How I could imagine that working would be that if no `pants.toml` existed, it would create one for you.

Not sure if that would run afoul of users using non-default config file names, but you could imagine disabling the behavior by looking at env vars.

Reply:
+1 to having the dedicated script so that people can inspect what they're about to run.